uml/uml.ttl

```diff
@@ -254,9 +255,27 @@ uml:postcondition rdf:type owl:ObjectProperty ;
 uml:ownedParameter rdf:type owl:ObjectProperty ;
          rdfs:subPropertyOf opil:compositionalProperty ;
          rdfs:domain uml:Behavior ;
-         rdfs:range uml:Parameter ;
+         rdfs:range uml:OrderedPropertyValue ;
          rdfs:label "parameters" .
 
+uml:OrderedPropertyValue rdf:type owl:Class ;
+         rdfs:subClassOf sbol:Identified ,
+    [ rdf:type owl:Restriction ; owl:allValuesFrom xsd:integer ; owl:onProperty uml:indexValue ;
+      owl:minCardinality "1"^^xsd:nonNegativeInteger ; owl:maxCardinality "1"^^xsd:nonNegativeInteger ] ,
+    [ rdf:type owl:Restriction ; owl:allValuesFrom xsd:anyURI ; owl:onProperty uml:propertyValue ;
+      owl:minCardinality "1"^^xsd:nonNegativeInteger ; owl:maxCardinality "1"^^xsd:nonNegativeInteger ] .
+
+uml:indexValue rdf:type owl:DatatypeProperty ;
+         rdfs:subPropertyOf opil:compositionalProperty ;
+         rdfs:domain uml:OrderedPropertyValue ;
+         rdfs:range xsd:integer ;
+         rdfs:label "index" .
+
+uml:propertyValue rdf:type owl:ObjectProperty ;
+         rdfs:subPropertyOf opil:compositionalProperty ;
+         rdfs:domain uml:OrderedPropertyValue ;
+         rdfs:range xsd:anyURI ;
```

Comment:
I believe that the range should to be an sbol:Identified rather than xsd:anyURI.

Reply:
It looks like you addressed the change on line 265, but not the linked change on line 276

Reply:
Oops, fixed that now.